docs/how-to-guides/provisioning-scripts.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Provisioning Scripts
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+Provisioning scripts can be used to override some of Rancher Desktop's internal processes. For example, scripts can be used to provide certain command line parameters to K3s, add additional mounts, increase ulimit value etc. This guide will explain how to set up your provisioning scripts for macOS, Linux, and Windows.
+
+## macOS & Linux
+On macOS and Linux, you can use lima override.yaml to write provisioning scripts. 
+
+- Create `override.yaml` file at below path
+
+<Tabs groupId="os">
+  <TabItem value="macOS" default>
+
+```
+~/Library/Application Support/rancher-desktop/lima/_config/override.yaml
+```
+
+  </TabItem>
+  <TabItem value="Linux">
+
+```
+~/.local/share/rancher-desktop/lima/_config/override.yaml
+```
+
+  </TabItem>
+</Tabs>
+
+- Write a provisioning script in the `override.yaml` file created in the previous step. For example, you can use the below script to create additional mounts.
```

Comment:
This is not really a provisioning script; it is just overriding/modifying a lima configuration setting.

Provisioning scripts are actual script (under the `provision` tag), that are not interpreted by Lima, but executed by a shell inside the VM.

Reply:
Modified the wording around examples provided to distinguish between a provisioning script and a simple lima configuration setting change.